Libraries/CustomComponents/NavigationExperimental/NavigationCardStackPanResponder.js

```diff
@@ -115,8 +111,12 @@ class NavigationCardStackPanResponder extends NavigationAbstractPanResponder {
       layout.width.__getValue();
 
     const positionMax = isVertical ?
-      RESPOND_POSITION_MAX_VERTICAL :
-      RESPOND_POSITION_MAX_HORIZONTAL;
+      props.gestureResponseDistance :
```

Comment:
I'm wondering if we should or not add a default value for `gestureResponseDistance` with **vertical** direction